blocks/server-side/additional-materials/additional-materials.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
+
+import cn from 'classnames';
+
+import styles from './additional-materials.module.css';
```

Comment:
I'd remove empty CSS file

Reply:
I put a style there for the subtitle, so now it's not empty